### PR TITLE
EDGE Update to pgbouncer, added sql plugins

### PR DESCRIPTION
**Pull request by silverdrake11**

### CODE REVIEW

File: bundle.yaml [hunk 1]
```diff
@@ -26,8 +26,17 @@ applications:
...
+      plugin_ltree_enable: true
+      plugin_intarray_enable: true
+      plugin_debversion_enable: true
+      plugin_pg_trgm_enable: true
+  pgbouncer-lds:
+    charm: pgbouncer
+    channel: 1/edge
+    options:
+      pool_mode: transaction
+      max_db_connections: 40
```
Comment (Perfect5th) [inline bundle.yaml:39]: Could you comment the reasoning for these settings please?

**Review by Perfect5th — CHANGES_REQUESTED** [review]

File: bundle.yaml (hunk 1 above)
Comment (silverdrake11) [reply bundle.yaml:39]: That number is how many connections are always open to be used pgbouncer for it's pool. It cannot be greater than the max connections of postgres which is 100 in this case.

Comment (silverdrake11) [reply bundle.yaml:39]: "You don't want default_pool_size to be equal to max_connections. That would leave no connections left over for things like monitoring, reporting, or debugging, or anything else. So back off a few. "

Comment (Perfect5th) [reply bundle.yaml:39]: Thanks!

**Review by Perfect5th — APPROVED** [review]

**Pull request merged**